assets/ros/core-monitor/style.css

```diff
@@ -8,7 +8,7 @@ table {
 }
 
 table thead {
-  background: #333;
+  background: var(--vscode-sideBarSectionHeader-background);
```

Comment:
I make use of the color from the background of sidebar header.
![image](https://user-images.githubusercontent.com/25241284/67437217-2cb55100-f5a5-11e9-9d00-e7164f140c9e.png)